### ISSUE TITLE
Hacky Hour Handbook

### ISSUE THREAD

**Issue by lexnederbragt** [uio-carpentry/studyGroup#18]
Just leaving this here as a potential Carpentry@UiO activity (together with the library and USIT):

https://github.com/amandamiotto/HackyHourHandbook/blob/master/Introduction.md

**Comment by annefou** [comment]
Yes. it is a very good idea. I am already part of this group but we haven't planned any hacky hours yet as part of our studyGroup (that was my initial plan).
